frontend/packages/dev-console/src/components/pipelines/resource-utils.ts

```diff
@@ -0,0 +1,57 @@
+import { get } from 'lodash';
+import {
+  PipelineResourceTask,
+  PipelineResourceTaskParam,
+  PipelineResourceTaskResource,
+} from '../../utils/pipeline-augment';
+
+type PipelineResourceTaskAlpha = PipelineResourceTask & {
+  spec: {
+    inputs?: {
+      params?: PipelineResourceTaskParam[];
+      resources?: PipelineResourceTaskResource[];
+    };
+    outputs?: {
+      resources?: PipelineResourceTaskResource[];
+    };
+  };
+};
+
+export type InputOutputResources = {
+  inputs?: PipelineResourceTaskResource[];
+  outputs?: PipelineResourceTaskResource[];
+};
+
+enum PATHS {
```

Comment:
Can't rely on the apiVersion for this release. Going with a brute force fetch concept.